docs/pages/blog/introducing-mui-base.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Introducing MUI Base
```

Comment:
Here are some ideas for potential titles:

- Introducing MUI Base: the headless alternative to Material UI 
  - quickly explains the primary value proposition of Base
- Customize with ease using MUI Base, our new headless UI library
  - centers the reader in the action while addressing the value proposition
- Introducing MUI Base: Material UI without Material Design 
  - may only make sense to existing users

We can hone in on the best description after we've decided on a title.

Reply:
Let's go with it!